### DISCUSSION TITLE
Improve L-BFGS

### DISCUSSION THREAD

**Discussion by HumbertHumbert7** [lululxvi/deepxde#1676] [General]
Hi, I need to achieve better precision using L-BFGS, how can I modify the source code and impose a smaller epsilon?

**Comment by lululxvi** [comment]
https://deepxde.readthedocs.io/en/latest/modules/deepxde.optimizers.html#deepxde.optimizers.config.set_LBFGS_options

**Comment by HumbertHumbert7** [comment]
Thank you Lu, but it doesn't cange nothing and if use the command
dde.config.set_default_float("float64")
as indicated, I have an error beacuse there are also some float32 that I am not able to fix.

**Reply by lululxvi** [reply]
It is possible that there are some data stored by default in float32, but this should be easy to fix.
